0.1.Copy data from Excel codes.R

```diff
@@ -1,79 +1,75 @@
-rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.
-gc() #free up memrory and report the memory usage.
+#rm(list = ls(all.names = TRUE)) #will clear all objects includes hidden objects.

```

Comment:
It's recommended to never explicitly delete things in the environment. What if you run your scripts in an R session where you had unsaved changes on which had been working for a long time?

You are right to try to create a clean environment for your script, but `rm(...)` won't do that anyway because it does not change what libraries are loaded nor what options are set.

For an almost-clean environment, run your script by `CTRL`-`SHIFT`-`k` (while holding down control and shift keys, type k).

For a truly clean environment, after you commit your changes, check out a new copy of your repository and run it in that folder.